monai/bundle/utils.py

```diff
@@ -9,10 +9,200 @@
+                "device": "@device"
+            }
+        ]
+    },
+    "dataset": {
+        "_target_": "Dataset",
+        "data": "$[{'image': i} for i in @datalist]",
+        "transform": "@preprocessing"
+    },
+    "dataloader": {
+        "_target_": "DataLoader",
+        "dataset": "@dataset",
+        "batch_size": 1,
+        "shuffle": false,
+        "num_workers": 0
+    },
+    "inferer": {
+        "_target_": "SimpleInferer"
+    },
+    "postprocessing": {
+        "_target_": "Compose",
+        "transforms": [
+            {
+                "_target_": "Activationsd",
+                "keys": "pred",
+                "softmax": true
+            },
+            {
+                "_target_": "AsDiscreted",
+                "keys": "pred",
+                "argmax": true
+            }
+        ]
+    },
+    "handlers": [
+        {
+            "_target_": "CheckpointLoader",
+            "_disabled_": "$not os.path.exists(@ckpt_path)",
+            "load_path": "@ckpt_path",
+            "load_dict": {"model": "@network"}
+        }
+    ],
+    "evaluator": {
+        "_target_": "SupervisedEvaluator",
+        "device": "@device",
+        "val_data_loader": "@dataloader",
+        "network": "@network",
+        "inferer": "@inferer",
+        "postprocessing": "@postprocessing",
+        "val_handlers": "@handlers"
+    }
+}"""
+
+
+def load_bundle_config(bundle_path: str, *config_names, **load_kw_args) -> Any:
+    """
+    Load the metadata and nominated configuration files from a MONAI bundle without loading the network itself.
+
+    This function will load the information from the bundle, which can be a directory or a zip file containing a
+    directory or a Torchscript bundle, and return the parser object with the information. This saves having to load
+    the model if only the information is wanted, and can work on any sort of bundle format.
+
+    Args:
+        bundle_path: path to the bundle directory or zip file
+        config_names: names of configuration files with extensions to load, should not be full paths but just name+ext
+        load_kw_args: keyword arguments to pass to the ConfigParser object when loading
+
+    Returns:
+        ConfigParser object containing the parsed information
+    """
+
+    from monai.bundle.config_parser import ConfigParser  # avoids circular import
+
+    parser = ConfigParser()
+
+    if not os.path.exists(bundle_path):
+        raise ValueError(f"Cannot find bundle file/directory '{bundle_path}'")
+
+    # bundle is a directory, read files directly
+    if os.path.isdir(bundle_path):
+        parser.read_meta(f=os.path.join(bundle_path, "configs", "metadata.json"), **load_kw_args)
+
+        for cname in config_names:
+            cpath = os.path.join(bundle_path, "configs", cname)
+            if not os.path.exists(cpath):
+                raise ValueError(f"Cannot find config file '{cpath}'")
+
+            parser.read_config(f=cpath, **load_kw_args)
```

Comment:
If calling `read_config` API multiple times, it will always refresh, please just call it once with all the files:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/bundle/config_parser.py#L247

Thanks.